code/modules/hydroponics/hydroponics.dm

```diff
@@ -551,10 +545,10 @@
 		adjustNutri(round(S.get_reagent_amount("fishwater") * 0.75))
 		adjustWater(round(S.get_reagent_amount("fishwater") * 1))
 
-	// Ambrosia Gaia produces earthsblood.
+	// You know what else has /datum/reagents, besides a beaker with earthsblood in it? That's right, ambrosia gaia!
```

Comment:
Despite saying so in #7919, Ambrosia Gaia composting is still a thing, since it contains /datum/reagents.

Since the grinder rounds up the reagents from the ingredients, using the grinder is FAR more viable. Example: Have a gaia branch with 1.4 earthsblood. Grinding gives 2 earthsblood. Now the question is, declare it as ""unintended mechanics depth"" or do something about it, like actually make gaia not compostable. You can still compost other plants though, and modify them to produce earthsblood. So, what the hell was the point of making gaia **NOT compostable**??